integration-cli/docker_cli_plugins_logdriver_test.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/docker/docker/pkg/integration/checker"
```

Comment:
This needs to be `github.com/docker/docker/integration-cli/checker` now, sorry 😅